### DISCUSSION TITLE
[Config Support]: Limit size of review clips

### DISCUSSION THREAD

**Discussion by saket424** [blakeblackshear/frigate#13232] [Ask A Question]
### Describe the problem you are having

Currently in a busy scene when there is no lull in the action, the size of the review clips becomes very large with too many events within one review clip

Is there a parameter to limit the duration of a review clip? Start a new review clip instead assuming that is feasible

### Version

0.15.0-47440e0f-tensorrt

### Frigate config file

```yaml
is there a limit review size setting in the config?
```

### Relevant log output

```shell
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

TensorRT

**Comment by NickM-27** [comment]
No, there is no option to control the max length
